app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -30,6 +30,38 @@ Locality-aware load balancing is enabled by default unlike its predecessor [loca
 - **`disabled`** – (optional) allows to disable locality-aware load balancing. When disabled requests are distributed 
 across all endpoints regardless of locality.
 
+{% if_version gte:2.5.x %}
+### Advanced LocalityAwareness
+Advanced locality-aware load balancing provides a more robust and straightforward method for balancing traffic within and across zones. This not only allows you to route traffic to a cross zone when the local zone service is unhealthy but also enables you to define traffic prioritization within the local zone and set cross-zone fallback priorities.
+
+#### LocalZone
+Local zone routing allows you to define traffic routing rules within a local zone, prioritizing data planes based on tags and their associated weights. This enables you to allocate specific traffic percentages to data planes with particular tags within the local zone. If there are no healthy endpoints within the highest priority group, the next priority group takes precedence. Locality awareness within the local zone relies on tags within inbounds, so it's crucial to ensure that the tags used in the policy are defined for the service (Dataplane object on Universal, Deployment annotations on Kubernetes).
+
+- **`localZone`** - (optional) allows to define load balancing priorities between dataplanes in the local zone. When not defined traffic is distributed equally to all endpoints within the local zone.
+  - **`affinityTags`** - list of tags and their weights based on which traffic is loadbalanced
+    - **`key`** - defines tag for which affinity is configured. The tag needs to be configured on the inbound of the service. In case of Kubernetes pod needs to have a tag and it should be propagated. On Universal user needs to define it on the inbound of the service. In case there so match, the dataplane match to the last group with the lowest weight.
```

Comment:
> should be propagated

what do you mean by that? propagated where, by what? propagated to all pods? if it's a strict requirement use `needs to be`, should implies that it's optional - https://datatracker.ietf.org/doc/html/rfc2119 

nitpick: comma missing between "case of Kubernetes, pod needs to"

> In case there so match

did you mean "in case there is no match"?

Reply:
Changed